project_one/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Project1</title>
+    <link rel="stylesheet" type="text/css" href="index.css" />
+  </head>
+  <body>
+    <header id="main-header">
+      <div class="container">
+        <h1>Project1</h1>
+      </div>
+    </header>
+
+    <nav id="navbar">
+      <div class="container">
+        <ul>
+          <li><a href="#Home">Home</a></li>
+          <!-- <li><a href="my_calc.html">Calculator Link</a></li> -->
+        </ul>
+      </div>
+    </nav>
+
+    <section>
+      <div class="container" id="receiver-div">
+        <p>This is section one:</p>
+        <button onclick="myFunction()" id="grading-element-one">Button1</button>
+      </div>
+    </section>
+
+    <section>
+      <div class="container">
+        <p>This is section two:</p>
+        <a id="grading-element-two" href="about.html">About</a>
+      </div>
+    </section>
+
+    <section>
+      <div class="container" id="grading-element-three">
+        <p>This is section three:</p>
+        <button onclick="myFunction2()">Button3</button>
+        <div id="grading-element-three-consequence" hidden>
+          awesome class
+        </div>
+      </div>
+    </section>
+
+    <section>
+      <div class="container form">
+        <p>This is section four:</p>
+        <form
+          action="http://localhost:3000/form"
+          method="post"
+          name="grading-element-four"
+        >
+          <select name="truthyVal">
+            <option value="testOne">testOne </option>
+            <option value="testTwo">testTwo</option>
+            <option value="testThree">testThree</option>
+          </select>
+          <input
+            name="text"
+            type="text"
+            placeholder="type here"
+            required
+            title="You must Type something"
+          />
+          <input type="radio" name="on-or-off" value="on" /> On
+          <input type="radio" name="on-or-off" value="off" />Off
+          <input type="submit" value="submit" />
+        </form>
+      </div>
+    </section>
+
+    <section>
+      <div class="container">
+        <p>This is section 5:</p>
+        <p id="grading-element-five" onmouseover="partFive()">
+          don't hover over this or else
+        </p>
+      </div>
+    </section>
+
+    <script>
+      function myFunction() {
+        let newThing = document.createElement("div");
+        newThing.innerText = "Add this to the DOM";
+        let receiverDiv = document.getElementById("receiver-div");
+        receiverDiv.appendChild(newThing);
+      }
+
+      function myFunction2() {
+        const x = document.getElementById("grading-element-three-consequence");
```

Comment:
you're close. But when i run `document.querySelector('#grading-element-three').click();` i do not see `grading-element-three-consequence`